src/main/java/crud/App.java

```diff
@@ -10,20 +14,52 @@
     private static final Manufacturer MANUFACTURER = new Manufacturer("Tesla", "USA");
     private static final Manufacturer MANUFACTURER_2 = new Manufacturer("Audi", "Germany");
     private static final Manufacturer MANUFACTURER_3 = new Manufacturer("Mercedes", "Germany");
-    private static final String MANUFACTURERS = "manufacturers";
+    private static final Driver DRIVER = new Driver("Ed", "123qwe222113");
+    private static final Driver DRIVER_2 = new Driver("John", "AAAEWWW222JHDSF8899");
+    private static final Driver DRIVER_3 = new Driver("BOB", "IEH709898SDF7689SDF231");
```

Comment:
don't use numbers in naming. User their real names DRIVER_ED , DRIVER_JOHN and so on